### DISCUSSION TITLE
How to Show a Button on selected Component . and customize the toolbar

### DISCUSSION THREAD

**Discussion by NoumanAhmad-Braqsol** [GrapesJS/grapesjs#3914] [Q&A]
I am actually what some that like that . See the blow image . Is it possible in grapesjs. ? Which approach would use . 
should i do it by changing the style of selected section or i can customize the toolbar . What should i do and how can get this interface. 
![Screenshot (52)](https://user-images.githubusercontent.com/72378934/139674748-b1e8661d-6fdb-4d82-9eb9-229a5e1337d8.png)

**Comment by artf** [comment]
I'd say that right now there is no easy way to accomplish that if not via workarounds (eg. customizing the view of each component on render). 
I'm planning to make the selected rectangle more customizable, which will allow a more stable way to accomplish such a case.
